### PR TITLE
"Diagnol difference array(Aizaz)"

### PR DESCRIPTION
**Pull request by Aizaz-tachyon**

### CODE REVIEW

File: src/com/trainee/aizaz/java/DiagnolDiffernceArray.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package com.trainee.aizaz.java;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class DiagnolDiffernceArray {
```
Comment (deepakmanikantak) [inline src/com/trainee/aizaz/java/DiagnolDiffernceArray.java:10]: Add Java Doc explaining the purpose of the class

File: src/com/trainee/aizaz/java/DiagnolDiffernceArray.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            int primarySum = 0;
+            int secondarySum = 0;
+
+            //finding the sum of each diagnol elements:
+            for (int i = 0; i < arr.length; i++) {
+                for (int j = 0; j < arr.length; j++) {
+                    if (i == j) {
+                        primarySum = primarySum + arr[i][j];
+
+                    }
```
Comment (deepakmanikantak) [inline src/com/trainee/aizaz/java/DiagnolDiffernceArray.java]: Add the comment  in more elaborate way

File: src/com/trainee/aizaz/java/DiagnolDiffernceArray.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+                    if ((i + j) == (n - 1)) {
+                        secondarySum = secondarySum + arr[i][j];
+
+
+                    }
+                }
```
Comment (deepakmanikantak) [inline src/com/trainee/aizaz/java/DiagnolDiffernceArray.java]: Remove unnecessary line breaks

File: src/com/trainee/aizaz/java/DiagnolDiffernceArray.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+            System.out.println("Please!! Enter a Valid Input");
+            System.out.println("Only Postive  Numners Allowed");
+
+        }
+
+
+    }
+}
+
```
Comment (deepakmanikantak) [inline src/com/trainee/aizaz/java/DiagnolDiffernceArray.java]: Remove unnecessary line breaks

**Review by deepakmanikantak — CHANGES_REQUESTED** [review]

**Review by Aizaz-tachyon — COMMENTED** [review]
i have added java doc and removed unnecessary spaces as well as  and added comment in more elaborate way

**Review by deepakmanikantak — APPROVED** [review]

**Pull request merged**